packages/kit-bg/src/vaults/impls/tron/Vault.ts

```
@@ -861,4 +870,145 @@
+              url: '/api/tronRent/uploadHash',
+              data: {
+                orderId,
+                fromHash: signedTx.txid,
+                signedData: JSON.parse(signedTx.rawTx),
+              },
+              params: {},
+            },
+          },
+        ],
+      });
+    if (resp && resp[0] && resp[0].success !== false && !resp[0].error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)